invenio_cli/commands/packages.py

```diff
@@ -37,6 +37,44 @@ def install_packages(packages):
 
         return steps
 
+    @staticmethod
+    def outdated_packages():
+        """Steps to show outdated packages.
+
+        It is a class method since it does not require any configuration.
+        """
+        prefix = ['pipenv', 'update']
+        cmd = prefix + ['--outdated']
+
+        steps = [
+            CommandStep(
+                cmd=cmd,
+                env={'PIPENV_VERBOSITY': "-1"},
+                message="Checking outdated packages..."
+            )
+        ]
+
+        return steps
+
+    @staticmethod
+    def update_packages(package):
+        """Steps to update all or one Python packages.
+
+        It is a class method since it does not require any configuration.
+        """
+        prefix = ['pipenv', 'update']
+        cmd = prefix + [package] if (package is not "") else prefix
```

Comment:
check it in the python console, but I'd swear that `""` string would evaluate to `False`. Meaning this could be changed to:
```suggestion
        cmd = prefix + [package] if not package else prefix
```